pkg/wkspace/builder.go

```diff
@@ -218,6 +223,6 @@ func mkdir(path string) error {
 }
 
 func manifestPath(repo string) string {
-	path, _ := filepath.Abs(filepath.Join(repo, "manifest.yaml"))
+	path, _ := filepath.Abs(pathing.SanitizeFilepath(filepath.Join(repo, "manifest.yaml")))
```

Comment:
Same here, `filepath.Abs()` might be undoing the changes that `pathing.SanitizeFilepath()` does.